src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -154,7 +154,7 @@ class RequestorStep extends React.Component {
                             lastName: this.props.getDefaultStateForField('lastName'),
                             street: this.props.getDefaultStateForField('requestorAddressStreet'),
                             city: this.props.getDefaultStateForField('requestorAddressCity'),
-                            state: this.props.getDefaultStateForField('requestorAddressState'),
+                            state: this.props.reimbursementAccountDraft.requestorAddressState || this.props.getDefaultStateForField('requestorAddressState'),
```

Comment:
This does not feel right. Can you please explain what problem is this solving?

Reply:
Can't remember exactly, but a bug was reported that should use the values from the draft when filling the state going to the previous screen and then back to the screen the State should show the draft, and the `getDefaultStateForField` function was always empty, so if there is something in the draft I try to use it

Reply:
This does not seem required. I think we fixed this already. Can you please double check from your side?

Reply:
Seems resolved